src/pipit/settings/__init__.py

```diff
@@ -19,3 +23,33 @@ def get_env(name, default=None):
 
 def get_env_bool(name, default=None):
     return get_env(name, default=default) == "True"
+
+
+def initialize_sentry(ingest_dsn: str, environment: str) -> None:
+    if ingest_dsn == "":
+        return
+
+    sentry_sdk.init(
+        dsn=ingest_dsn,
+        environment=environment,
+        debug=False,
+        traces_sample_rate=1.0,
+        integrations=[
+            DjangoIntegration(
+                # Nothing interesting here
+                middleware_spans=False,
+                # Wagtail produces a lot of signals
+                signals_spans=False,
+            )
+        ],
+        release="0.1.0",
```

Comment:
Should we sync this release with the releases on [github](https://github.com/ZEnMo/Holon-webapp/releases)?

Reply:
I put the field because it's mandatory.

I don't think it matters, we know which version is running on which environment.